includes/admin/class-wc-admin-menus.php

```diff
@@ -147,10 +148,14 @@ public function status_menu() {
 	 * Addons menu item.
 	 */
 	public function addons_menu() {
+		add_menu_page( __( 'Marketplace', 'woocommerce' ), __( 'Marketplace', 'woocommerce' ), 'manage_woocommerce', 'wc-addons', null, null, '58' );
```

Comment:
Do we have an icon for this menu?

Reply:
We removed the menu for now, so the icon won't be necessary.